src/resolvers/mutations.ts

```diff
@@ -365,7 +364,7 @@ const sanitize = (ctx: FullContext, model: EntityModel, data: Entity) => {
     }
 
     if (isEndOfDay(field) && data[key]) {
```

Comment:
```suggestion
    if (isEndOfDay(field) && typeof data[key] === 'string') {
```
so you do not need ` as string` below, wdyt?

Reply:
we discussed